fleetconfig-controller/internal/controller/v1beta1/addon.go

```diff
@@ -404,6 +386,17 @@ func handleAddonEnable(ctx context.Context, spoke *v1beta1.Spoke, addons []v1bet
 			enableErrs = append(enableErrs, fmt.Errorf("failed to enable addon: %v, output: %s", err, string(out)))
 			continue
 		}
+		// TODO - do this natively with clusteradm once https://github.com/open-cluster-management-io/clusteradm/issues/501 is resolved.
```

Comment:
These two sentences seem contradictory. Why will this still need to happen manually once clusteradm supports specifying addon config(s) when enabling addons?